metrics/web/test/auth/presentation/widgets/strategy/google_sign_in_option_strategy_test.dart

```diff
@@ -32,5 +48,21 @@ void main() {
 
       verify(notifier.signInWithGoogle()).called(1);
     });
+
+    test(
+        ".getWidgetAppearance() returns the login option button theme if not in the loading state",
+        () {
+      final actualTheme = strategy.getWidgetAppearance(metricsTheme, false);
+
+      expect(actualTheme, loginThemeData.loginOptionButtonStyle);
```

Comment:
```suggestion
      final expectedStyle = loginThemeData.loginOptionButtonStyle;

      final actualStyle = strategy.getWidgetAppearance(metricsTheme, false);

      expect(actualStyle, equals(expectedStyle));
```
Would it be more AAA-like? Anyway, I suppose we should add `equals` and change namings to style.